test/sql/returning/returning_delete.test

```diff
@@ -105,6 +105,8 @@ INSERT INTO table2 VALUES (1, 1, 1), (2, 2, 2), (3, 3, 3), (100, 100, 100), (200
 statement ok
 INSERT INTO table3 VALUES (1, 4, 4), (2, 6, 7), (8, 8, 8);
 
+mode skip
```

Comment:
Is the skip intended to be here still?

Reply:
hmm, yes and no. I skipped it because the query requires sorting, but the returning statement doesn't allow sorting. I've changed the order now, but if it fails again we should find a different way to test it